pins/drivers.py

```diff
@@ -206,10 +234,20 @@ def default_title(obj, name):
     import pandas as pd
 
     if isinstance(obj, pd.DataFrame):
+        try:
```

Comment:
this might play really nicely with the changes you've made in #263 👀 What if we merge that PR first and then refactor `geopandas` dataframes to be part of [`_get_df_family`](https://github.com/nathanjmcdougall/pins-python/blob/2c2e5029c300631822d1326d5b94cde895b4294b/pins/drivers.py#L27)?

Reply:
Totally agree.